db/models/stigData.ts

```diff
@@ -49,22 +51,29 @@ export class StigData extends Model<InferAttributes<StigData>, InferCreationAttr
   declare AssessmentItems?: NonAttribute<AssessmentItem[]>;
   declare EvaluationItems?: NonAttribute<EvaluationItem[]>;
 
-  // declare SystemId: ForeignKey<System["id"]>;
-  // declare StigDatumId: ForeignKey<StigData["id"]>;
-
   declare Stigs?: NonAttribute<Stig[]>;
-  // declare SystemId: ForeignKey<System["id"]>;
-  // declare StigDatumId: ForeignKey<StigData["id"]>;
   declare StigIdents?: NonAttribute<StigIdent[]>;
   declare Overrides?: NonAttribute<Override[]>;
+  declare StigResponsibilities?: NonAttribute<StigResponsibility[]>;
+  declare StigReferences?: NonAttribute<StigReference[]>;
+
+  declare addStigResponsibility: BelongsToManyAddAssociationMixin<StigResponsibility, number>;
```

Comment:
Maybe group the functions with the variables?  Or at least put them in the same order as the variables are defined.